apps/src/netsim/NetSimLocalClientNode.js

```diff
@@ -307,44 +307,6 @@ NetSimLocalClientNode.prototype.getMyRouter = function () {
 };
 
 /**
- * Destroy the local node; performs appropriate clean-up leading up to
- * node destruction.
- * @param {!NodeStyleCallback} onComplete
- */
-NetSimLocalClientNode.prototype.destroy = function (onComplete) {
```

Comment:
This custom `destroy` method goes away entirely, so the base class `NetSimEntity.destroy` gets called instead.  The additional cleanup is all handled on the server anyway now.